Makefile

```diff
@@ -0,0 +1,94 @@
+
```

Comment:
Btw, this repo uses `justfile`, so it would be great to use that and modify if needed

Reply:
I did it for myself. I accidentally push. I will delete

Reply:
Sure, no problem. If you find something that is useful for all dev of this package, feel free to modify justfile